src/api/app/views/webui/projects/status/show.html.haml

```diff
@@ -76,7 +76,7 @@
                     = summary
                     %br
                 %td{ data: { package_name: package['name'] } }
-                  = render partial: 'status_comment', locals: { package_name: package['name'], editable: package['firstfail'].present?,
+                  = render partial: 'webui/project/status_comment', locals: { package_name: package['name'], editable: package['firstfail'].present?,
```

Comment:
That partial is used in other places? If not, we could move it to the new directory

Reply:
yes, its used in another view under project. So I would keep it linked instead of have a copy of that for now.